app/src/Event/EventController.php

```diff
@@ -1149,6 +1150,74 @@ public function editTracks($friendly_name)
         );
     }
 
+    /**
+     * Upload Data from CSV for this event
+     * @todo Validate & Process uploaded cSV
+     * @param string $friendly_name
+     */
+    public function eventImportCsv($eventSlug)
+    {
+        $config = $this->application->config('oauth');
+        $request = $this->application->request();
+
+        /** @var FormFactoryInterface $factory */
+        $factory = $this->application->formFactory;
+        $form    = $factory->create(new EventImportFormType());
+
+        //use EventFormImportType to validate the CSV is valid
+        //We possibly want to do some extensive data checking
+        //BEFORE we blindly throw data from here to the API
```

Comment:
"possibly" is an understatement here, we need to add validation here as there is no validation being done in the api, so it must be done here

Reply:
:(